src/main/java/org/jabref/logic/net/URLDownload.java

```diff
@@ -171,6 +172,21 @@ public String getMimeType() {
         return "";
     }
 
+    /**
+     * Check the connection by using the HEAD request.
+     * UnirestException can be thrown for invalid request.
+     * @return the status code of the response
+     */
+    public int checkConnection() {
+        Unirest.config().setDefaultHeader("User-Agent", "Mozilla/5.0 (Windows; U; WindowsNT 5.1; en-US; rv1.8.1.6) Gecko/20070725 Firefox/2.0.0.6");
+        try {
+            int statusCode = Unirest.head(source.toString()).asString().getStatus();
+            return statusCode;
+        } catch (UnirestException e) {
```

Comment:
(obsolte with the refactoring to Boolean return value)

Why this statement? Can't the `try..catch` constrcut just be removed and a ` throws UnirestException` added to the method?